doc/whats_new/v0.20.rst

```diff
@@ -225,6 +225,9 @@ Changelog
 - |Fix| :func:`datasets.fetch_openml` to retry downloading when reading
   from local cache fails. :issue:`12517` by :user:`Thomas Fan <thomasjpfan>`.
 
+- |Fix| :func:`datasets.load_sample_images` returns images with a deterministic
```

Comment:
Seems that Joel labelled the issues as 0.20.3.

Reply:
I think I labelled it as 0.20.3 by mistake. The doctest isn't in 0.20. I'd rather this fix in 0.21